control-plane/grpc/proto/v1/node/node.proto

```diff
@@ -17,21 +17,15 @@ message Node {
   optional NodeState state = 3;
 }
 
-// Node cordon information
-message NodeCordon {
-  // Node cordon label.
-  repeated string label = 2;
-}
-
 message NodeSpec {
   // Node identification
   string node_id = 1;
   // Endpoint of the io-engine instance (gRPC)
   string endpoint = 2;
   // Node labels.
   common.StringMapValue labels = 3;
-  // Cordon information.
-  NodeCordon cordon = 4;
+  // Drain state
+  optional DrainState drainstate = 4;
```

Comment:
can we improve the `DrainState` naming, maybe `CordonDrainState`, or `SchedulingState`, something else?

Reply:
Changed to CordonDrainState throughout (I considered CordonOrDrainState, but that's getting too long)